### ISSUE TITLE
store ges scale as gaussian ply.

### ISSUE THREAD

**Issue by archershot** [ajhamdi/ges-splatting#9]
render_laplacian has modify the gaussian scale from 'pc.get_scaling' to 'pc.get_scaling * pc.get_shape'.
https://github.com/ajhamdi/ges-splatting/blob/c05fc7dbb22e270a5a6f490e7040adac4af02c96/gaussian_renderer/__init__.py#L150C9-L150C47

so when store laplacian module to gaussian ply the raw scale should be
"scale = self.scaling_inverse_activation(self.get_scaling * self.get_shape).detach().cpu().numpy()"
instead of 
"scale = (self._scaling * self.get_shape).detach().cpu().numpy()"

https://github.com/ajhamdi/ges-splatting/blob/c05fc7dbb22e270a5a6f490e7040adac4af02c96/scene/laplacian_model.py#L217C6-L217C72

As
